src/ansys/aedt/core/application/analysis.py

```diff
@@ -2402,3 +2232,273 @@ def number_with_units(self, value, units=None):
+            Dictionary of nominal variations with values.
+
+        """
+        available_variables = self.__available_variables()
+        return {k: v.expression for k, v in list(available_variables.items())}
+
+    @property
+    def nominal_w_values_dict(self):
+        """Nominal independent with values in a dictionary.
+
+        Returns
+        -------
+        dict
+            Dictionary of nominal independent variations with values.
+
+        References
+        ----------
+        >>> oDesign.GetChildObject('Variables').GetChildNames
+        >>> oDesign.GetVariables
+        >>> oDesign.GetVariableValue
+        >>> oDesign.GetNominalVariation
+        """
+        warnings.warn(
+            "`nominal_w_values_dict` is deprecated. " "Use `nominal_values` method instead.", DeprecationWarning
+        )
+        families = {}
+        for k, v in list(self._app.variable_manager.independent_variables.items()):
+            families[k] = v.expression
+
+        return families
+
+    @property
+    def variables(self):
+        """Variables.
+
+        Returns
+        -------
+        list of str
+            List of names of independent variables.
+        """
+        warnings.warn(
+            "`variables` is deprecated. " "Use `variable_manager.independent_variable_names` method instead.",
+            DeprecationWarning,
+        )
+        return self._app.variable_manager.independent_variable_names
+
+    @property
+    def nominal_w_values(self):
+        """Nominal independent with values in a list.
+
+        Returns
+        -------
+        list
+            List of nominal independent variations with expressions.
+
+        References
+        ----------
+        >>> oDesign.GetChildObject('Variables').GetChildNames()
+        >>> oDesign.GetVariables
+        >>> oDesign.GetVariableValue
+        >>> oDesign.GetNominalVariation
+        """
+        warnings.warn("`nominal_w_values` is deprecated. " "Use `nominal_values` method instead.", DeprecationWarning)
+        families = []
+        for k, v in list(self._app.variable_manager.independent_variables.items()):
+            families.append(k + ":=")
+            families.append([v.expression])
+        return families
+
+    @property
+    def nominal_w_values_dict_w_dependent(self):
+        """Nominal independent and dependent with values in a dictionary.
+
+        Returns
+        -------
+        dict
+            Dictionary of nominal independent and dependent variations with values.
+
+        References
+        ----------
+        >>> oDesign.GetChildObject('Variables').GetChildNames
+        >>> oDesign.GetVariables
+        >>> oDesign.GetVariableValue
+        >>> oDesign.GetNominalVariation"""
+        warnings.warn("`nominal_w_values_dict_w_dependent` is deprecated.", DeprecationWarning)
+        families = {}
+        for k, v in list(self._app.variable_manager.variables.items()):
+            families[k] = v.expression
+
+        return families
```

Comment:
```suggestion
       families = {
    k: v.expression
    for k, v in self._app.variable_manager.variables.items()
}

return families
```
@Samuelopez-ansys and here.

Reply:
This is deprecated, so no changes are needed. This will be removed in future versions.